app/models/authority.rb

```diff
@@ -220,7 +220,8 @@ def morph_url
 
   def morph_feed_url_for_date_range(start_date, end_date)
     query = CGI.escape("select * from `data` where `date_scraped` >= '#{start_date}' and `date_scraped` <= '#{end_date}'")
-    "https://api.morph.io/#{morph_name}/data.json?query=#{query}&key=#{ENV['MORPH_API_KEY']}"
+    key = CGI.escape ENV['MORPH_API_KEY'] || ""
```

Comment:
Hey @equivalentideas ! It does seem reasonable to throw an exception when the API key isn't set at all, but in this case the key was wrong and rake did throw an error. In any case, let me if I can help with adding an exception. :smile: